### PR TITLE
[fix](statistics)Cast min/max in partition stats table to double for numeric column to get correct table level min/max value.

### PR DESCRIPTION
**Pull request by Jibing-Li**
Min/max value in partition statistics table are of type String, we need to cast numeric type to double when we merge partition stats to get table stats, otherwise, the table min/max value is incorrect because they are in alphabet order.

Currently, this function is closed by default. Will open it later with more test cases.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Jibing-Li** [comment]
run buildall

**Comment by Jibing-Li** [comment]
run performance

**Review by morningman — DISMISSED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/statistics/ExternalAnalysisTask.java [hunk 1]
```diff
@@ -88,6 +88,8 @@ protected Map<String, String> buildSqlParams() {
             params.put("type", col.getType().toString());
         }
         params.put("lastAnalyzeTimeInMs", String.valueOf(System.currentTimeMillis()));
+        params.put("min", col.getType().isNumericType() ? "CAST(min AS DOUBLE)" : "min");
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/statistics/ExternalAnalysisTask.java:91]: to col's type is better?

**Comment by Jibing-Li** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Comment by Jibing-Li** [comment]
run buildall

**Review by morningman — APPROVED** [review]

**Comment by Jibing-Li** [comment]
run feut

**Pull request merged**